src/execution/operator/order/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-add_library_unity(duckdb_operator_order OBJECT physical_order.cpp)
+add_library_unity(duckdb_operator_order
+                  OBJECT
+                  physical_order.cpp
+                  physical_order_limit.cpp)
```

Comment:
Also please change this to `physical_top_n.cpp` and change the operators themselves to be called `PhysicalTopN` and `LogicalTopN`.

Reply:
Agree.